core/trashcan.ts

```diff
@@ -68,7 +68,8 @@ export class Trashcan extends DeleteArea implements IAutoHideable,
    * The trashcan flyout.
    * @internal
    */
-  flyout: IFlyout|null = null;
+  // Flyout is always initialized in the constructor.
```

Comment:
Nit, because construction and initialization do different things for the flyout: it's created *and* initialized in the constructor, and the creation is the part that the non-nullability addresses.

Reply:
I looked again at `trashcan` and I actually had the wrong solution. `flyout` may not be initialized in the constructor if the `maxTrashcanItems` is 0. So my previous fix was wrong.

Instead on the line that eslint complained about, I changed the `?` to a `!` because in that case we know `flyout` is initialized (since we are doing the same check on the options).